usecase/parser/ibcmsg/ibcmsg.go

```diff
@@ -21,15 +22,20 @@ import (
 	"github.com/crypto-com/chain-indexing/usecase/parser/utils"
 )
 
+var allowedClientStateTypes = []string{
+	"/ibc.lightclients.tendermint.v1.ClientState",
+	"/ibc.lightclients.solomachine.v2.ClientState",
+}
+
 func ParseMsgCreateClient(
 	msgCommonParams event.MsgCommonParams,
 	txsResult model.BlockResultsTxsResult,
 	msgIndex int,
 	msg map[string]interface{},
 ) []command.Command {
 	clientStateType := msg["client_state"].(map[string]interface{})["@type"]
-	if clientStateType != "/ibc.lightclients.tendermint.v1.ClientState" {
-		// TODO: SoloMachine and Localhost LightClient
+
+	if c, ok := clientStateType.(string); ok && !slice.ContainString(allowedClientStateTypes, c) {
```

Comment:
I think if we assume the `clientStateType` is always a string, then we could replace `ok` with `_`.

What do you think?